Source/Scene/GroundPrimitive.js

```diff
@@ -0,0 +1,655 @@
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default false
+         */
+        interleave : {
+            get : function() {
+                return this._primitive.interleave;
+            }
+        },
+
+        /**
+         * When <code>true</code>, the primitive does not keep a reference to the input <code>geometryInstances</code> to save memory.
+         *
+         * @memberof GroundPrimitive.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default true
+         */
+        releaseGeometryInstances : {
+            get : function() {
+                return this._primitive.releaseGeometryInstances;
+            }
+        },
+
+        /**
+         * When <code>true</code>, each geometry instance will only be pickable with {@link Scene#pick}.  When <code>false</code>, GPU memory is saved.         *
+         *
+         * @memberof GroundPrimitive.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default true
+         */
+        allowPicking : {
+            get : function() {
+                return this._primitive.allowPicking;
+            }
+        },
+
+        /**
+         * Determines if the geometry instances will be created and batched on a web worker.
+         *
+         * @memberof GroundPrimitive.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default true
+         */
+        asynchronous : {
+            get : function() {
+                return this._primitive.asynchronous;
+            }
+        },
+
+        /**
+         * When <code>true</code>, geometry vertices are compressed, which will save memory.
+         *
+         * @memberof GroundPrimitive.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default true
+         */
+        compressVertices : {
+            get : function() {
+                return this._primitive.compressVertices;
+            }
+        },
+
+        /**
+         * Determines if the primitive is complete and ready to render.  If this property is
+         * true, the primitive will be rendered the next time that {@link GroundPrimitive#update}
+         * is called.
+         *
+         * @memberof GroundPrimitive.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         */
+        ready : {
+            get : function() {
+                return this._ready;
+            }
+        },
+
+        /**
+         * Gets a promise that resolves when the primitive is ready to render.
+         * @memberof GroundPrimitive.prototype
+         * @type {Promise.<GroundPrimitive>}
+         * @readonly
+         */
+        readyPromise : {
+            get : function() {
+                return this._readyPromise;
+            }
+        }
+    });
+
+    GroundPrimitive._maxHeight = 9000.0;
```

Comment:
Oh right.  Submit an issue or add this to a roadmap since the fillrate optimizations will be important (we can also use less conservative top - and bottom - caps with a min/max terrain query).